### DISCUSSION TITLE
GUI scripting

### DISCUSSION THREAD

**Discussion by GundraHarshini** [The-OpenROAD-Project/OpenROAD#6425] [General]
In GUI scripting window while typing the commands its not showing the available commands , previously its showing like if i type read_ it shows the related commands like read_lef , read_sdc.. what to do to get back to it

**Comment by maliberty** [comment]
There is no recent code change nor any way to disable completion that I know of.  Does it persist if you quite and reopen?

**Reply by gadfort** [reply]
If you right click in the text field I think there is an option to disable auto complete. You can also try opening OpenROAD with `-no_settings`

**Reply by GundraHarshini** [reply]
Thank you, its working
